.github/workflows/packaging.yml

```diff
@@ -0,0 +1,22 @@
+name: Packaging
+
+on:
+  release:
+    types: [released]
+
+
+jobs:
+  winget:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup winget-create
+        shell: powershell
+        run: |
+          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+      - name: Update manifest and submit
+        shell: powershell
+        run: |
+            $version = ${{ github.ref }}.Substring(1)
+            wingetcreate.exe update --id GitHub.cli --url https://github.com/cli/cli/releases/download/v${version}/gh_${version}_windows_amd64.msi --version ${version} --out .\manifest\ --submit True --token ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
            wingetcreate.exe update GitHub.cli --url https://github.com/cli/cli/releases/download/v${version}/gh_${version}_windows_amd64.msi --version ${version} --out .\manifest\ --submit True --token ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Can we pull down a specific build of wingetcreate instead of pulling the latest version? Breaking changes like this might happen in future releases as well.